src/main/java/com/parsing/parsers/pdf/parsing/model/Column.java

```diff
@@ -0,0 +1,14 @@
+package com.parsing.parsers.pdf.parsing.model;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+
+import java.awt.*;
+
+@Data
```

Comment:
We decide use @Getter and @Setter instead @Data in the project.

Reply:
I think that for DTOs it's fine, For Entities it's better to use getter, setter and other annotations implicitly